practice_app/pages/api/getTranslation.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios')
+const { checkTranslationResponse } = require('./getTranslationModule')
+
+export default async (req, res) => {
+    // req object contains whatever information we can get from the request that comes from the client (from the browser)
+    // res object is used tow write our reponse
+    const { query, method } = req
+
+    switch (method) {
+        case 'GET':
+            {
+                console.log(query)
+                let target = query.countryCode == null ? '' : query.countryCode
+                target = target.toLowerCase()
+                const text = query.text == null ? '' : query.text
+                let sourceLang = query.sl == null ? '' : query.sl
+
+                await axios
+                    .get(
+                        `https://translate.googleapis.com/translate_a/single?client=gtx&sl=${sourceLang}&tl=${target}&dt=t&q=${text}`,
+                    )
+                    .then(response => {
+                        const values = checkTranslationResponse(response, sourceLang, target, text)
+                        const responseText = values.translation
+                        sourceLang = values.sourceLang
+                        res.statusCode = 200
+                        res.json({
+                            answered: 'yes',
```

Comment:
Is it ever possible to have `answered == 'no'` ? If so what is the point? 

Also would have been better to have it as a boolean instead.

Reply:
"answered" changed to boolean. It is now false when the input is invalid.